src/Orchard.Cms.Web/project.json

```diff
@@ -15,7 +15,13 @@
     "Microsoft.AspNetCore.Diagnostics": "1.1.0",
     "Microsoft.AspNetCore.Mvc.ViewFeatures": "1.1.0",
     "System.Dynamic.Runtime": "4.3.0",
-    "Microsoft.Extensions.Localization.Abstractions": "1.1.0"
+    "Microsoft.Extensions.Localization.Abstractions": "1.1.0",
+
+    "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design": {
+      "version": "1.1.0-*",
```

Comment:
Consider pinning the tool to `1.1.0-preview4-final` since the rest of the packages are pinned. You really don't need to specify `"target": "package"` (especially since you aren't building the tool locally and referencing it from your global.json)